addons/website/static/src/builder/plugins/options/avatars_option.xml

```diff
@@ -0,0 +1,135 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+<t t-name="website.AvatarsOption">
+    <BuilderRow label.translate="Size">
+        <BuilderRange
+            styleAction="'--avatars-size'"
+            min="2"
+            max="6"
+            step="0.1"
+            displayRangeValue="false"
+            unit="'rem'"
+        />
+    </BuilderRow>
+    <BuilderRow label.translate="Order">
+        <BuilderSelect>
+            <BuilderSelectItem classAction="''">Last on top</BuilderSelectItem>
+            <BuilderSelectItem classAction="'o_first_on_top'">First on top</BuilderSelectItem>
+        </BuilderSelect>
```

Comment:
Depending on what you do about my comment on the `updateZindexFunction`, this option would therefore become this :arrow_down: :slightly_smiling_face: 
```xml
<BuilderSelect action="'avatarsChangeOrder'">
    <BuilderSelectItem actionParam="''">Last on top</BuilderSelectItem>
    <BuilderSelectItem actionParam="'o_first_on_top'">First on top</BuilderSelectItem>
</BuilderSelect>
```